lib/draft/schema.json

```diff
@@ -343,6 +357,192 @@
             }
           ]
         },
+        "normalizedPayload": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/endDevicePayloadCodec/definitions/scriptOutput"
+            },
```

Comment:
This should be in `normalizePayloadOutput`

So there's:

- `normalizedPayload` with only the normalized payload properties

- `normalizePayloadOutput` which is the output of the normalize payload script (that has an `allOf` of script output as well as `normalizedPayload`)

- `uplinkNormalizer` under `endDevicePayloadCodec` that takes `decodeOutput` as input and `normalizePayloadOutput` as output

Reply:
Okey! Thanks for your comments, I'll make changes accordingly.